src/sth_server.js

```diff
@@ -560,7 +561,8 @@
                   contextElement = contextResponses[i].contextElement;
                   attributes = contextElement.attributes;
                   for (var j = 0; j < attributes.length; j++) {
-                    if (!attributes[j].value || isNaN(attributes[j].value)) {
+                    if (!attributes[j].value ||
+                      (isNaN(attributes[j].value) && typeof(attributes[j].value) !== 'string')) {
```

Comment:
Relying on the "string" type is not totally safe, I mean, you can receive a float having a "string" type (because of an error when defining the entities). Thus, I think the best is relying on `isNaN` function. Or, as you propose, if the `occur` aggregation is set, then always use the `toString` method. Anyway, this is my own personal opinion :) I did not specify the feature, just food for think ;)